app/globals.css

```
@@ -1333,3 +1357,365 @@
+  to {
+    transform: none;
+    opacity: 1;
+  }
+}
+.rctn-roll {
+  animation: rctn-roll 0.28s cubic-bezier(0.22, 1, 0.36, 1) both;
+}
+
+/* 좌우 흩어짐(--rctn-dx)과 회전(--rctn-rot)은 탭마다 컴포넌트가 인라인으로 준다 */
+@keyframes rctn-burst {
+  from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)